pkg/util/propagatedversion/propagatedversion.go

```diff
@@ -19,7 +18,7 @@ This file may have been modified by The KubeAdmiral Authors
 are Copyright 2023 The KubeAdmiral Authors.
 */
 
-package util
```

Comment:
Would it be better to move this directory under `pkg/controllers/sync`? Since it is and will probably continue to not be used by other controllers?